### PR TITLE
#165413096 Integrate with coveralls

### PR DESCRIPTION
**Pull request by AJAkimana**
Description
=======
This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product

Type of change
=======
N/A

How has it been tested
=======
N/A

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413096](https://www.pivotaltracker.com/n/projects/2326984/stories/165413096)

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env"]
+}
```
Comment (macphilips) [inline .babelrc]: Add new line

**Review by macphilips — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -6,11 +6,19 @@
   "scripts": {
     "test": "nyc mocha --require @babel/register \"tests/**/*.js\"",
     "lint": "eslint --ignore-path .gitignore .",
-    "lint:fix": "npm run lint --fix"
+    "lint:fix": "npm run lint --fix",
+    "migrate": "",
```
Comment (macphilips) [inline package.json]: Remove this line. it's of no use

**Review by macphilips — CHANGES_REQUESTED** [review]

**Review by Niyitangasam — CHANGES_REQUESTED** [review]
I am not able to run this locally.
image

**Review by Niyitangasam — APPROVED** [review]

File: .babelrc (hunk 1 above)
Comment (AJAkimana) [reply .babelrc]: > Add new line

:all-done

File: package.json (hunk 2 above)
Comment (AJAkimana) [reply package.json]: > Remove this line. it's of no use
I am On it

Comment (AJAkimana) [reply package.json]: > Remove this line. it's of no use
it's done

**Review by ericnyirimana — APPROVED** [review]
It works great, good work

File: .gitignore [hunk 3]
```diff
@@ -34,9 +34,16 @@ node_modules
...
 #Report directory
 .nyc_output
 
 package-lock.json
 # dot env file
 .env
+
+yarn.lock
+
+package-lock.json
```
Comment (macphilips) [inline .gitignore]: We already have package-lock.json defined in this file kindly remove this

File: .gitignore [hunk 4]
```diff
@@ -34,9 +34,16 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+# Environnement
+.env
```
Comment (macphilips) [inline .gitignore:38]: We already have .env defined in this file kindly remove this

**Review by macphilips — CHANGES_REQUESTED** [review]

**Pull request merged**